test/gaudi_opts/test_global_converter_maps.py

```diff
@@ -18,9 +18,9 @@
 # limitations under the License.
 #
 
-import os
 
-from Gaudi.Configuration import *
+from Gaudi.Configuration import INFO, DEBUG
+import sys
```

Comment:
```suggestion
import os
```

`sys` should not be necessary, but `os` should?

Reply:
Neither are needed